roles/os_firewall/tasks/firewall/firewalld.yml

```diff
@@ -1,7 +1,8 @@
 ---
 - name: Install firewalld packages
-  package: name=firewalld state=present
-  when: not openshift.common.is_containerized | bool
+  package:
+    name: firewalld
+    state: present
```

Comment:
wondering if this should have a conditional for `not openshift.common.is_atomic | bool`. otherwise, lgtm